### DISCUSSION TITLE
Why I can't get benefits?

### DISCUSSION THREAD

**Discussion by justin1106** [community/community#135500] [GitHub Education] [answered]
### Hi everyone,

Hello, I'm justin1106. I applied for the GitHub student developer pack and it was approved on August 10th. The announcement says I can benefit within 3 days, but today is the 3rd day. Will I be able to benefit within today?

**Comment by davevad93** [comment, marked as answer]
Hi @justin1106 ,

Waiting times are approximate and may vary depending on different factors, such as the volume of requests. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @justin1106 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135145

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by justin1106** [reply]
Thank you for your comment. I will wait more, but I want to know where is the status label.

**Reply by davevad93** [reply]
You're welcome @justin1106 , you can find the status label on the application page [here](https://education.github.com/discount_requests/application), scroll down a bit and look on the right side where it says "***You've already submitted X requests***".

**Comment by justin1106** [comment]
Oh, I found it. Thank you so much.

2024년 8월 12일 (월) 오후 9:37, Davide Presti ***@***.***>님이 작성:
